types/lumino-js-sdk.d.ts

```diff
@@ -105,13 +121,7 @@ declare module 'lumino-js-sdk' {
          * @return {Promise} Channels - Returns a Observable that, when fulfilled, will either return an Array with the
          * channels or an Error with the problem. The channels obtained are only open.
          */
-        getChannel({
-            tokenAddress,
-            partnerAddress
-        }: {
-            tokenAddress: string;
-            partnerAddress: string;
-        }): Promise<any>;
+        getChannel(identifier: ChannelIdentifier): Promise<any>;
```

Comment:
As the above comment, i think this could be `ChannelRequest` or `ChannelFilter` or something else

Reply:
but it's neither of those things, it's literally the identifier of a channel: a token address and a partner address. it comes all the way down from the BL

Reply:
this discussion was settled outside of this platform: with heavy hearts, we've settled on "params"